activity.py

```diff
@@ -31,7 +31,7 @@
     gi.require_version('Gst', '1.0')
     from gi.repository import Gst
     _HAS_SOUND = True
-except:
+except BaseException:
```

Comment:
This does not indicate why is the `try` clause being used.
Use `ImportError` instead of `BaseException`